src/components/Navigation.jsx

```diff
@@ -1,17 +1,19 @@
 import React from 'react';
 import { NavLink } from 'react-router-dom';
 
+import './Navigation.css';
+
 const Navigation = () => {
   return (
-    <div>
+    <div className="navigation">
       <ul>
         <li>
-          <NavLink to="/list-view" activeStyle={{ fontWeight: 'bold' }}>
+          <NavLink to="/list-view" activeClassName="active">
             ListView
           </NavLink>
         </li>
         <li>
-          <NavLink to="/add-view" activeStyle={{ fontWeight: 'bold' }}>
+          <NavLink to="/add-view" activeClassName="active">
```

Comment:
I like how activeClassName works here! 👍🏾

Reply:
Thanks! We found it by searching about the activeStyle property you added originally, so thanks for the inspiration! 😁 🎉

Reply:
This is nice, as it is not good to use `activeStyle={{ fontWeight: 'bold' }}`. It is considered inline styling and if you had a lot of styling then the code would not be so readable. Good job 👍